docs/learningLab/introToLit/network.md

```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 2
+---
+
+# 2. How does the network work?
+
+![image](../../../static/img/networkOverview.png)
+
+The Lit network operates through a distributed federation of nodes, where each node plays a vital role in key generation, storage, consensus, and the execution of Lit Actions. However, what sets Lit apart is its reliance on threshold cryptography, which ensures that no single node holds complete control or "executive authority" over the network.
```

Comment:
Could remove the first sentence ('storage' is somewhat misleading) and just lead off with 'lit is powered by threshold cryptography, meaning no single node holds complete control or "executive authority" over the network' before diving in